Node/src/botbuilder-teams.d.ts

```diff
@@ -887,6 +887,8 @@ export type SigninStateVerificationHandlerType = (event: builder.IEvent, query:
 export type FileConsentCardResponseHandlerType = (event: builder.IEvent, response: IFileConsentCardResponse, callback: (err: Error, result: any, statusCode?: number) => void) => void;
 export type TaskModuleFetchHandlerType = (event: builder.IEvent, request: ITaskModuleInvokeRequest, callback: (err: Error, result: ITaskModuleResponseOfFetch, statusCode?: number) => void) => void;
 export type TaskModuleSubmitHandlerType = (event: builder.IEvent, request: ITaskModuleInvokeRequest, callback: (err: Error, result: ITaskModuleResponseOfSubmit, statusCode?: number) => void) => void;
+export type ComposeExtensionFetchForActionCommandHandlerType = (event: builder.IEvent, request: ComposeExtensionQuery, callback: (err: Error, result: ITaskModuleResponseOfFetch, statusCode?: number) => void) => void;
```

Comment:
for both fetch and submit, the response could be task module, or compose extension, so you would need a union type of both.